contracts/src/challenge/ChallengeManager.sol

```diff
@@ -291,7 +291,14 @@ contract ChallengeManager is DelegateCallAware, IChallengeManager {
         override
         returns (uint256)
     {
-        return challenges[challengeIndex].current.timeLeft;
+        ChallengeLib.Challenge storage challenge = challenges[challengeIndex];
+        uint256 timeLeft = challenge.current.timeLeft;
+        uint256 timeSinceLastMove = challenge.timeUsedSinceLastMove();
+        if (timeLeft > timeSinceLastMove) {
+            return timeLeft - timeSinceLastMove;
+        } else {
+            return 0;
+        }
```

Comment:
Looks like this method isn't currently used anywhere as far as I can see. Maybe better to just remove it for simplicity/minimal contract size

Reply:
I've removed this, and separately added a new ResolveNodes staker strategy which I think is useful